hw/ip_templates/racl_ctrl/rtl/racl_ctrl.sv.tpl

```diff
@@ -78,21 +78,15 @@ module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
   assign alert[0]  = shadowed_update_err;
   assign alert[1]  = reg_intg_error | shadowed_storage_err;
 
-  assign alert_test = {
-    reg2hw.alert_test.fatal_fault.q &
-    reg2hw.alert_test.fatal_fault.qe,
-    reg2hw.alert_test.recov_ctrl_update_err.q &
-    reg2hw.alert_test.recov_ctrl_update_err.qe
-  };
+  assign alert_test[0] = reg2hw.alert_test.fatal_fault.q & reg2hw.alert_test.fatal_fault.qe;
```

Comment:
Definitely not for this PR, but I wonder whether it would make sense to define an enum with the two indices, so this ends up `alert_test[FatalAlert] = ...` or similar?

Reply:
> Definitely not for this PR, but I wonder whether it would make sense to define an enum with the two indices, so this ends up `alert_test[FatalAlert] = ...` or similar?

Definitely, we use the same approach for assigning incoming interrupts/alerts in the chip environment.

Reply:
Created https://github.com/lowRISC/opentitan/issues/27398 to track.